index.html

```diff
@@ -0,0 +1,88 @@
+          <span class="dateTag">Posted on 15 Dec 2013 / Desserts</span>
+          <a href="" class="print">
+            <img src="images/print.jpg">
+            <span class="printText">Print</span>
+          </a>
+        </section>
+        <img src="images/mainImage.jpg" class="mainImage">
+
+        <p>For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stirs, set aside and let cool completely.</p><br />
+
+        <p>Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p><br />
+
+        <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p><br />
+
+        <section class="checklist">
+          <ul class="grid_5 prefix_1">
+            <li><input type="checkbox"> 1 1/2 cups milk</option></li>
+            <li><input type="checkbox"> 1/2 cup mascarpone</option></li>
+            <li><input type="checkbox"> 1/2 tsp pink salt</option></li>
+            <li><input type="checkbox"> 1 lb Black Mission Figs</option></li>
+            <li><input type="checkbox"> 1/2 cup brown sugar</option></li>
+            <li><input type="checkbox"> 2-4 tbsp water</option></li>
+          </ul>
+          <ul class="checklist2">
+            <li><input type="checkbox"> 1 1/2 cups heavy cream</option></li>
+            <li><input type="checkbox"> 1/2 granulated sugar</option></li>
+            <li><input type="checkbox"> 2 egg yolks</option></li>
+            <li><input type="checkbox"> 1 lemon, juiced</option></li>
+            <li><input type="checkbox"> 2 tbsp butter</option></li>
+            <li><input type="checkbox"> 1 cup honey roasted pecans, roughly chopped</option></li>
+          </ul>
+        </section>
+
+        <section class="authorSection">
+          <img src="images/lineBreak.jpg" class="lineBreak">
+          <section class="authorDetails grid_11 alpha">
+            <img src="images/author.jpg" class="authorImage">
+            <h2 class="authorName">Vanessa Stevenson</h2>
+            <p class="authorBio">Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
+          </section>
+          <img src="images/shareButton.jpg" class="shareRecipe">
+        </section>
+      </article>
+
+      <footer class="grid_16 alpha">
```

Comment:
It's probably not necessary to add the footer to the grid since it will span the entire parent regardless.